src/codeu/chat/server/Server.java

```diff
@@ -42,245 +42,251 @@
 
 public final class Server {
 
-  private interface Command {
-    void onMessage(InputStream in, OutputStream out) throws IOException;
-  }
+	private interface Command {
+		void onMessage(InputStream in, OutputStream out) throws IOException;
+	}
+
+	private static final Logger.Log LOG = Logger.newLog(Server.class);
 
-  private static final Logger.Log LOG = Logger.newLog(Server.class);
+	private static final int RELAY_REFRESH_MS = 5000; // 5 seconds
```

Comment:
Please clean up these unchanged lines so they are actually unchanged to git.

Adhere to the style of the code base, which in this case uses spaces instead of tabs. I suggest running [google-java-format](https://github.com/google/google-java-format) to take care of these cases.